prowler/providers/aws/services/cognito/cognito_service.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+################## Cognito
+class Cognito(AWSService):
+    def __init__(self, audit_info):
+        super().__init__("cognito-idp", audit_info)
+        self.user_pools = []
```

Comment:
Please set this as a map instead of a list, using the `user_pool` ARN as key of the dict.